--- ggv-1.1.95/configure.in~	Thu Oct  3 21:35:26 2002
+++ ggv-1.1.95/configure.in	Thu Oct  3 21:35:26 2002
@@ -140,10 +140,6 @@
 AC_SUBST(OAF_LIBS)
 AC_SUBST(OAF_CFLAGS)
 
-AC_SUBST(CFLAGS)
-AC_SUBST(CPPFLAGS)
-AC_SUBST(LDFLAGS)
-
 AC_OUTPUT([
 Makefile
 ggv.spec
@@ -154,7 +150,6 @@
 help/C/Makefile
 help/it/Makefile
 help/ja/Makefile
-pixmaps/Makefile
 src/Makefile
 src/stock/Makefile
 bonobo/Makefile
