diff -urN aa/ggv-2.5.99/configure.in ggv-2.5.99/configure.in
--- aa/ggv-2.5.99/configure.in	2004-03-05 16:32:43.000000000 +0100
+++ ggv-2.5.99/configure.in	2004-03-06 12:06:41.240966200 +0100
@@ -87,7 +87,7 @@
 dnl **************************************************
 dnl * internationalization support
 dnl **************************************************
-ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id is it ja kn ko lt lv mk ml mn ms nl nn no pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi zh_CN zh_TW"
+ALL_LINGUAS="af am ar az be bg bn ca cs cy da de el en_CA en_GB es et eu fa fi fr ga gl gu he hi hr hu id is it ja kn ko lt lv mk ml mn ms nb nl nn pa pl pt pt_BR ro ru sk sl sq sr sr@Latn sv ta th tr uk vi zh_CN zh_TW"
 
 GETTEXT_PACKAGE=ggv
 AC_SUBST(GETTEXT_PACKAGE)
